api/controllers/case.js

```diff
@@ -195,6 +133,7 @@ function getUpdatedCase(user, params, newCase, oldCase) {
     cond("hidden", as.boolean);
     cond("original_language", as.text);
     cond("post_date", as.date);
+  } else {
```

Comment:
doesn't look like we need this else block

Reply:
Agreed.